Tweak.xm

```diff
@@ -1,173 +1,129 @@
-CGPoint dockPoint;
+#include <UIKit/UIKit.h>
+#include "SpringBoardHome.h"
+
+BOOL isEnabled;
 BOOL shouldHide;
+BOOL initialized;
 
-@interface SBDockView : UIView
-@end
-
-@interface SBRootFolderView : UIView
-@property (nonatomic, assign, readwrite) NSUInteger dockEdge;
-@property (nonatomic,readonly) SBDockView *dockView; 
-@end
-
-%group ios13
-
-%hook SBRootFolderView
-
--(void)updateIconListIndexAndVisibility:(BOOL)arg1{
-    if (@available(ios 13, *)) {
-        self.dockEdge = 1;
-        UIEdgeInsets safeAreaInsets = [[[UIApplication sharedApplication] keyWindow] safeAreaInsets];
-        if (safeAreaInsets.bottom != 0) {
-            self.dockView.frame = CGRectMake(0, 50, self.dockView.frame.size.width, self.dockView.frame.size.height);
-            if (!dockPoint.x) {
-                dockPoint = CGPointMake(self.dockView.center.x, (self.dockView.center.y + 50));
-            }
-            self.dockView.center = dockPoint;
-        }
-        %orig;
-    } else {
-		%orig;
-	}
-}
+#define safeAreaInsets [[[UIApplication sharedApplication] keyWindow] safeAreaInsets]
 
+#define kIconController [%c(SBIconController) sharedInstance]
+#define kRootFolderController [kIconController _rootFolderController]
+#define kIconModel [kIconController model]
+#define kDockView kRootFolderController.rootFolderView.dockView
```

Comment:
I use several macros here to keep the length of lines down. It also makes some sections of code far easier to read.